playbooks/roles/redis-exporter/README.md

```diff
@@ -0,0 +1,14 @@
+redis-exporter
+==============
+
+Extends prometheus_redis_exporter_role to be used in Ocim.
```

Comment:
@shimulch, @0x29a, imho, this role should be a generic redis-exporter role not specific to Ocim and it should be possible to configure and use it for Ocim by overriding the variables that this role provides. CC @giovannicimolin